mangaki/mangaki/management/commands/compare.py

```diff
@@ -30,8 +30,7 @@ class Experiment(object):
     results = {}
     algos = None
     def __init__(self, PIG_ID=None):
-        # self.algos = [MangakiALS(20), MangakiSVD(20), MangakiKNN(20), MangakiZero()]
```

Comment:
As a side note, maybe it would be time for those to be command-line arguments :-)
(Let's do that in another patch, though).